cypress/e2e/49_adminPaginationNumber.cy.ts

```diff
@@ -0,0 +1,44 @@
+describe('Super Admin Pagination Bounties List', () => {
+  let activeUser = 'alice';
+
+  const bounty: Cypress.Bounty = {
+    title: 'Pagination Bounty',
+    category: 'Web development',
+    coding_language: ['Typescript', 'Javascript', 'Lightning'],
+    description: 'This is available',
+    amount: '123',
+    tribe: 'Amazing Org Tribe',
+    estimate_session_length: 'Less than 3 hour',
+    estimate_completion_date: '09/09/2024',
+    deliverables: 'We are good to go man'
+  };
+
+  beforeEach(() => {
+    cy.login(activeUser);
+    cy.wait(1000);
+  });
+
+  it('Should create 25 bounties and verify they are listed in the admin page', () => {
+    for (let i = 1; i <= 25; i++) {
+      const updatedBounty = { ...bounty, title: `Pagination Bounty ${i}` };
+      cy.create_bounty(updatedBounty);
+      cy.wait(1000);
+    }
+
+    cy.visit('http://localhost:3007/admin');
+    cy.wait(3000);
+
+    for (let i = 25; i <= 6; i--) {
```

Comment:
@AbdulWahab3181 please change the loop max length to 22, then search for the last 2 on the next page, it broke for me at some point.

Reply:
@elraphty Addressed